lib/fog/compute/kubevirt/models/vms.rb

```diff
@@ -52,11 +52,11 @@ def create(args = {})
           end
           
           volumes = []
-
+          volume_name = vm_name.split('.')[0]
```

Comment:
Why not to use something we implemented in the apb => https://github.com/ansibleplaybookbundle/import-vm-apb/blob/master/bin/run-v2v.sh#L95 ?